src/components/Toolbar/Toolbar.jsx

```diff
@@ -1,11 +1,16 @@
 import React from "react";
 import { Link, NavLink, withRouter } from "react-router-dom";
-
 import homeLogo from "../../images/homeLogo.png";
-
-import { FaSearch } from "react-icons/fa";
+import { FaSearch, FaUser } from "react-icons/fa";
+import { useUser } from "../../contexts/UserContext";
```

Comment:
is there absolute paths or something we can use for these? seems like a recipe for disaster if the folder structure changes for some reason

Reply:
I don't actually know the answer to that. I just know most times whatever IDE you're in can handle changing the relative paths and file names when you move or change a file.

Reply:
With TS you could use some `paths`, but with I'm JS not sure. But all modern IDEs shouldn't have a problem with updating imports when e.g. you move a file.

Reply:
You can do that in the jsconfig.js file by setting the baseUrl property to point to the src directory in the compiler options. The only problem is that I'm not sure how that's going to affect the tests, because we're already doing something similar in the .env file. That being said, I believe the .env file is going to be deprecated in favor of setting the baseUrl in the next major release of React.

Reply:
So are there any steps i should take on this PR?

Reply:
No. Maybe open an issue for that.